uber/site_sections/preregistration.py

```diff
@@ -8,33 +7,57 @@ def to_sessionized(attendee, group):
     else:
         return Charge.to_sessionized(attendee)
 
+
 def check_prereg_reqs(attendee):
-    if attendee.badge_type == PSEUDO_DEALER_BADGE and not attendee.cellphone:
+    if attendee.badge_type == c.PSEUDO_DEALER_BADGE and not attendee.cellphone:
         return 'Your phone number is required'
-    elif attendee.amount_extra >= SHIRT_LEVEL and attendee.shirt == NO_SHIRT:
+    elif attendee.amount_extra >= c.SHIRT_LEVEL and attendee.shirt == c.NO_SHIRT:
         return 'Your shirt size is required'
 
+
 def check_dealer(group):
```

Comment:
Yeah, that's one of the things on my TODO list; having "preregistration" validations handled by the new API and also somehow marked separately from the regular ones.  I haven't gotten to it yet since MAGStock didn't need it, but it's definitely something we need to do.